.gitignore

```diff
@@ -136,3 +136,4 @@ dmypy.json
 /bin/
 /obj/
 /tmp/
+out
```

Comment:
Newline?

Reply:
We should add `blobStorage` as well, due to the change in volumes in `docker-compose.override.yml`

Reply:
fixed